content/chainguard/chainguard-images/authenticating.md

```diff
@@ -0,0 +1,95 @@
+---
+title: "Authenticating to Chainguard Registry"
+type: "article"
+description: "A guide on authenticating to the Chainguard Registry to get images"
+date: 2023-03-21T15:10:16+00:00
+lastmod: 2023-03-21T15:10:16+00:00
+draft: false
+images: []
+menu:
+  docs:
+    parent: "chainguard-images"
```

Comment:
TODO: make this registry after rebasing on https://github.com/chainguard-dev/edu/pull/503